tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from beebop.app import app as flask_app
+
+
+@pytest.fixture()
+def app():
+    app = flask_app
+    app.config.update({
+        "TESTING": True,
```

Comment:
It enables testing mode. According to the documentation this causes:

> Exceptions are propagated rather than handled by the the app’s error handlers. Extensions may also change their behavior to facilitate easier testing